tests/test_generate_hooks.py

```diff
@@ -120,6 +134,32 @@ def test_oserror_hooks(mocker):
 
 
 @pytest.mark.usefixtures('clean_system', 'remove_additional_folders')
+def test_popen_includes_hooks_in_python_path(mocker):
+    err = OSError()
+    prompt = mocker.patch('subprocess.Popen')
```

Comment:
Test that the Popen receives a copy of the current env with the **hooks** folder included in PYTHONPATH